### PR TITLE
DOC: improves docstrings.

### LINKED ISSUE #857: Decide on docstring style for sphinx-apidoc / autodoc
Intelmq is implemented in python3 thus I consider it agreeable that we use www.sphinx-doc.org as processor to generate our code documentation. (As opposed to the overview documentation which is independent and currently done in some Markdown variant.)

There seem be several styles for docstrings around (see http://www.sphinx-doc.org/en/stable/ext/autodoc.html#module-sphinx.ext.autodoc)
 * reStructuredText (the default of sphinx)
 * NumPy style
 * Google style

http://www.sphinx-doc.org/en/stable/ext/napoleon.html#module-sphinx.ext.napoleon is the sphinx plugin adding the two additional styles and comparing them. As we are python3, **my proposal is:**

 * Declare **module-sphinx.ext.napoleon "google style"** the default autodoc style for intelmq
 * Declare **python3 type annotations** as the way types SHOULD be documented

### PR DESCRIPTION
**Pull request by bernhardreiter**
* Removes ERRORs and WARNINGs when running sphinx-doc with napoleon,
   see #910 . The changes are done in a minimal way to adhere to the
   chosen style and reSt and remove the warning. So there is still a lot
   room for improvement.

resolves #857

### CODE REVIEW

File: intelmq/bots/parsers/anubisnetworks/parser.py [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
 
-_geo_env_remote_addr.country_code       => source.geolocation.cc
-_geo_env_remote_addr.country_name       => source.geolocation.country
-_geo_env_remote_addr.region             => source.geolocation.region
-_geo_env_remote_addr.city               => source.geolocation.city
-_geo_env_remote_addr.asn                => source.geolocation.asn
-_geo_env_remote_addr.asn_name           => source.geolocation.as_name
-_geo_env_remote_addr.longitude          => source.geolocation.longitude
-_geo_env_remote_addr.longitude          => source.geolocation.longitude
-_geo_env_remote_addr.ip + netmask       => source.network
```
Comment (wagner-certat) [inline intelmq/bots/parsers/anubisnetworks/parser.py]: Did you accidentally remove these lines?

Comment (bernhardreiter) [inline intelmq/bots/parsers/anubisnetworks/parser.py]: Yes, I've tried the github conflict resolution interface for the first time and somehow messed it up. Good catch, thanks!

Comment (bernhardreiter) [inline intelmq/bots/parsers/anubisnetworks/parser.py]: Small formattos also fixed.

**Comment by wagner-certat** [comment]
merged

**Pull request closed without merging**